core/domain/takeout_service.py

```diff
@@ -59,6 +61,13 @@ def export_data_for_user(user_id):
                                 model export policy>
         }.
     """
+    user_settings = user_services.get_user_settings(user_id)
+    # TODO(#10358): Remove this check and allow takeout for profiles (Learner
```

Comment:
Drop the TODO in this case, since it is not actually clear yet whether we want to do this at all / whether there is a feasible way to do this. (The issue should suffice; it is basically a feature request.)

Reply:
Okay, dropped the comment now.